### PR TITLE
Update email-settings-android-enterprise.md

### PR DESCRIPTION
**Pull request by msbemba**
Added a pre req as Chrome app needs to be  installed in the work profile to enable modern auth for Gmail Per issue#https://github.com/MicrosoftDocs/memdocs/issues/2891 and support case

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by PRMerger17** [comment]
@msbemba : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/memdocs/issues/2891

File: memdocs/intune/configuration/email-settings-android-enterprise.md [hunk 1]
```diff
@@ -43,6 +43,7 @@ As an Intune administrator, you can create and assign email settings to Android
 
 - Create an [Android Enterprise email device configuration profile](email-settings-configure.md) > **Personally-owned work profile**.
 - Or, create an [app configuration policy](../apps/app-configuration-policies-use-android.md).
+- Install Chrome App in the work profile to enable modern auth for Gmail.
```
Comment (JohanFreelancer9) [inline memdocs/intune/configuration/email-settings-android-enterprise.md:46]:
```suggestion
- Install the Chrome app in the work profile to enable modern auth for Gmail.
```
```suggestion
- Install Chrome App in the work profile to enable modern auth for Gmail.
```

Comment (JohanFreelancer9) [inline memdocs/intune/configuration/email-settings-android-enterprise.md:46]:
```suggestion
- Install the Chrome app in the work profile to enable modern auth for Gmail.
```
```suggestion
- Install Chrome App in the work profile to enable modern auth for Gmail.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by PRMerger5** [comment]
@msbemba : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@MandiOhlinger Please review and signoff this PR. Thanks

**Comment by MandiOhlinger** [comment]
Holding off on this PR. Requested more information in https://github.com/MicrosoftDocs/memdocs/issues/2891.

**Comment by PRMerger20** [comment]
@MandiOhlinger : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger4** [comment]
@MandiOhlinger : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by MandiOhlinger — APPROVED** [review]

**Comment by MandiOhlinger** [comment]
@msbemba Thank you for making these changes. I updated the text. #sign-off

**Pull request merged**